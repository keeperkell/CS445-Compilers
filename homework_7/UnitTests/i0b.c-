## C-S22
main() begin 
     begin 
        int rr[20];
        output(*rr);
     end 
 end 
