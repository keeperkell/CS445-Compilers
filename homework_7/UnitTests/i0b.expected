WARNING(5): Variable 'rr' may be uninitialized when used here.
Number of warnings: 1
Number of errors: 0
------------------------------------
Loading file: UnitTests/i0b.tm
20
Bye.
